context.go

```diff
@@ -967,6 +997,9 @@ func (c *Context) Pow(d, x, y *Decimal) (Condition, error) {
 
 // Quantize adjusts and rounds v as necessary so it is represented with
 // exponent exp and stores the result in d.
+// TODO(mjibson): "adjusts"?
```

Comment:
If you have a specific suggestion I welcome it. This function is pretty hard to describe. There are runnable examples in the docs, though, that make it easy to understand.